addons/mail/static/src/widgets/discuss/discuss.js

```diff
@@ -320,7 +320,7 @@ const DiscussWidget = AbstractAction.extend({
      * @private
      */
     _onClickMarkAllAsRead() {
-        this.env.models['mail.message'].markAllAsRead(this.domain);
+        this.env.models['mail.message'].markAllAsRead(this.discuss.domain);
```

Comment:
Isn't there no `domain` field on discuss model? This should be `stringifiedDomain` instead?